m4_sessions/1801-inning/group_6.md

```diff
@@ -17,3 +17,55 @@ Finding The Right Fit
 - Samuel Singer
 
 @emhickmann
+
+# Group Six
+
+### Pitch: 
+Are you one of those students who says you'll just take any job after Turing?? So were we! Come to our session to get a little more insight into your options, and how to narrow down your search to companies who are the right fit for YOU.
+
+### Session Title: 
+What's your favorite flavor of shit sandwich (and does it come with an olive?)
+
+https://markmanson.net/life-purpose
+
+
+### Session Outcomes:
+- Understand what drives you, where you do your best work, what makes you unique
+- Understand the pros and cons in the  with working at a start-up, mid-size, large company, agency vs. consultancy and how you fit within each
+
+### Session Outline:
+#### 10 mins- Introduction:
+1- 2 min self reflection:
+- what motivates you
+- what do you value
+- when's a time you did your best work?
+- when's a time you weren't able to do your best work? why? 
+Introductions-
+ - Name, Mod
+ -Share on Reflection
+ 
+ General share on what company people want to work in before we break out. 
+ #### 50 mins Company Culture Details
+ 10 mins per area with 5 mins after to discuss anything that stood out. any of the points raised surprising? how do your values align with those points?
+ 
+- Companies(Sam):
+    Corporations vs. Small to Mid
+- Consultancy/agency vs. product company (Aurora):
+- Startups(Emily):
+- Remote (Erika):
+
+#### 10 mins Wrap-up and reflect
+were any of the points raised surprising? how do your values align with those points?
+10 min of do yout think differently now about what you want to do? why?  (whole group after 3 min think time. is anyone considering something that they werent previously?
+but how does this align with what you thrive in in a working environment?
```

Comment:
I might add, 
"How can you use your time at Turing to continue to explore these options?"  "Does this session make you want to explore new/different companies?"